test/acceptance/features/steps/steps.py

```diff
@@ -190,6 +206,78 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
 
 
 # STEP
+is_serverless_operator_running_step = u'Openshift Serverless Operator is running'
```

Comment:
do we really need this extra var, or we could apply the approach similar to others steps:

```

@given(u'Openshift Serverless Operator is running')
def given_serverless_operator_is_running(context)
```

Reply:
We could apply the same for other steps as well. As this var that holds the description of the step can be in `given` or any other keywords like `then`